src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1577,11 +1577,21 @@ def _get(self, entity, entnum, item1, it1num, item2, it2num):
             self._get_lock = False
 
         if getresponse.type == 0:
-            raise ValueError(
-                "This is either an invalid get request, or MAPDL is set"
-                " to the wrong processor (e.g. on BEGIN LEVEL vs."
-                " POST26)"
+            self._log.debug(
+                "The 'grpc' get method seems to have failed. Trying old implementation for more verbose output."
             )
+            try:
+                out = self.run("*GET,__temp__," + cmd)
+            except MapdlRuntimeError:
+                # Get can thrown some errors, in that case, they are caught in the default run method.
+                raise
+            else:
+                # Here we catch the rest of the errors and warnings
+                raise ValueError(out)
+            finally:
+                # deleting parameter
+                self.run("__temp__=", mute=False)
```

Comment:
Not necessary. There's a small chance that this fails, and at best this incurs a roundtrip MAPDL <--> PyMAPDL call.

Reply:
What do you mean roundtrip? Issuing ``__temp__=`` should not return anything to MAPDL, right?

Reply:
We just don't need to clear the parameter, that's all. It does return, it just returns None, so it's blocking until this is complete. If we didn't have a blocking call, we could rapidly get into a race condition and kill the server.

Reply:
Ahhmm.. OK. I'm not fan of leaving vars and files around. But given the stability issues we have, I agree on this. (Also leading underscored parameters should not be shown).